rosetta/services/transaction_parser.go

```diff
@@ -201,9 +209,36 @@ func (tp *transactionsParser) createOperationsFromPreparedTx(tx *data.Transactio
 	return operations
 }
 
-func computeTxFee(eTx *data.FullTransaction) string {
-	fee := big.NewInt(0).SetUint64(eTx.GasPrice)
-	fee.Mul(fee, big.NewInt(0).SetUint64(eTx.GasLimit))
+func (tp *transactionsParser) createRosettaTxFromInvalidTx(eTx *data.FullTransaction) *types.Transaction {
```

Comment:
Please add TODO and Jira task that we have to update this for the mainnet upgrade.